src/engine/test/source/api/test/session_test.cpp

```diff
@@ -0,0 +1,178 @@
+#include <api/test/sessionManager.hpp>
+#include <gtest/gtest.h>
+#include <testsCommon.hpp>
+
+using namespace api::sessionManager;
+
+const auto& SESSION_NAME {"sessionDummy"};
+const auto& POLICY_NAME {"policyDummy"};
+const auto& FILTER_NAME {"filterDummy"};
+const auto& ROUTE_NAME {"routerDummy"};
```

Comment:
Personally, I think that adding all the files of assets directory is not the best approach to test this.
It would make more sense if we start using it to test another modules as default Dummy assets. 
We should discuss this on a subsequent iteration.